.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
-        - E.g. download the tarballs with `download_tarball`, and fix them up with `fix_azdo_tarball` if necessary.
-1.  - [ ] [Internal] Rename tarball and smoke-test prereqs to human-friendly names. (Can use `rename_tarball_inner_dir`.)
+        - If the baseline change is due to the change in patch version, make
+          sure there's an issue tracking fixing the instability, or start
+          tracking it.
+        - If in doubt, talk to the team. Someone might have already encountered
+          a similar diff in the past.  The Source-Build Teams channel is a good
+          place to start the conversation, or the dotnetADTeam DL.
+1. - [ ] Add team members crummel, LoganBussell, and msimons as reviewers to
+         the PR. Get acceptance.
+      - Note: Don't merge the PR.  This is just used to generate the pre-embargo-date
+        tarball for our partners.  This will be ported to the public repo
+        later and become the official release.
+1. - [ ] Download the artifacts from the centos71_Online_BaseTarball PR
+         validation build onto a machine with Bash.
+      - From the "Artifacts" page, navigate to "Tarball centos71_Online_BaseTarball"
+        and download the `base_tarball_*.tar.gz` and `tarball_*-smoke-test-prereqs.tar.gz`
+        files.
+      - The smoke-test prereqs are platform-independent but should be taken
+        from this non-portable build, *not* CentOS7 Offline Portable.
+      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh>
+        as a library.
+        - E.g. download the tarballs with `download_tarball`, and fix them up
+          with `fix_azdo_tarball` if necessary.
+1. - [ ] Rename tarball and smoke-test prereqs to human-friendly names.
+         (Can use `rename_tarball_inner_dir`.)
       - Refer to previous release's email threads for naming pattern.
-1.  - [ ] [Internal] Upload tarballs and smoke-test prereqs to blob storage.
-1.  - [ ] [Internal] Send the tarball to partners. Include info about how certain we are that this will be the final Microsoft build.
-      - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-1.  - [ ] SYNC POINT: Wait for Microsoft build release.
-      - Wait for dotnet/announcements post. <kbd>Watch</kbd> the repo to get emails so you don't have to poll.
+1. - [ ] Upload tarballs and smoke-test prereqs to the `dotnetclimsrc` blob
+         storage under the `source-build` container in `red-hat/3.1/` under a
+         new folder named for the runtime and SDK version.  The access key for
+         this blob storage is in the `EngKeyVault` vault in the "Dotnet
+         Engineering Services" subscription.
+1. - [ ] Complete manual validation steps
```

Comment:
It feels like there should be a checkpoint where a source-build team member signs off.  Maybe that is concurrent w/the manual platform tests.  Having it earlier may be beneficial though just to give a once-over on the changes.  This would be especially beneficial on the first couple of runs.

Reply:
There's an earlier step:
> Add team members crummel, LoganBussell, and msimons as reviewers to the PR. Get acceptance.

on line 151.